### PR TITLE
[SDR-273] BE : 가게의 리뷰 목록 조회 API 구현

### PR DESCRIPTION
**Pull request by jinan159**
### ❗️ 이슈 번호

[SDR-273]

<br><br>

### 📝 구현 내용

- 가게의 리뷰 목록 조회 API 구현
  - 리뷰 조회 API 는 `StoreController` 에서 처리
  - 리뷰 조회 기능은 `ReviewService.searchReviewsByStoreId` 에서 처리
<br><br>

### 💡 참고 자료

image

<details>
<summary>가게 리뷰 조회 API 문서</summary>
<div class="sect2">
<h3 id="_가게_리뷰_조회get_apistoresstoreidreviewsafteraftersizesize"><a class="link" href="#_가게_리뷰_조회get_apistoresstoreidreviewsafteraftersizesize">가게 리뷰 조회(GET /api/stores/{storeId}/reviews?after={after}&amp;size={size})</a></h3>
<div class="sect3">
<h4 id="_request_24"><a class="link" href="#_request_24"><code>Request</code></a></h4>
<div class="sect4">
<h5 id="_request_24_http_request"><a class="link" href="#_request_24_http_request">HTTP request</a></h5>
<div class="listingblock">
<div class="content">
<pre class="highlightjs highlight nowrap"><code data-lang="http" class="language-http hljs"><span class="hljs-keyword">GET</span> <span class="hljs-string">/api/stores/1/reviews?after=0&amp;size=1</span> HTTP/1.1
<span class="hljs-attribute">Accept</span>: application/json
<span class="hljs-attribute">Content-Type</span>: application/json
<span class="hljs-attribute">Host</span>: sikdorak.herokuapp.com</code></pre>
</div>
</div>
</div>
<div class="sect4">
<h5 id="_request_24_path_parameters"><a class="link" href="#_request_24_path_parameters">Path parameters</a></h5>
<table class="tableblock frame-all grid-all stretch">
<caption class="title">Table 1. /api/stores/{storeId}/reviews</caption>
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top">Parameter</th>
<th class="tableblock halign-left valign-top">Description</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>storeId</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">가게 아이디</p></td>
</tr>
</tbody>
</table>
</div>
<div class="sect4">
<h5 id="_request_24_request_parameters"><a class="link" href="#_request_24_request_parameters">Request parameters</a></h5>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top">Parameter</th>
<th class="tableblock halign-left valign-top">Description</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>size</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">페이지 크기</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>before</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">이전 페이지를 가리키는 커서</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>after</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">다음 페이지를 가리키는 커서</p></td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="sect3">
<h4 id="_response_24"><a class="link" href="#_response_24"><code>Response</code></a></h4>
<div class="sect4">
<h5 id="_response_24_http_response"><a class="link" href="#_response_24_http_response">HTTP response</a></h5>
<div class="listingblock">
<div class="content">
<pre class="highlightjs highlight nowrap"><code data-lang="http" class="language-http hljs">HTTP/1.1 <span class="hljs-number">200</span> OK
<span class="hljs-attribute">Vary</span>: Origin
<span class="hljs-attribute">Vary</span>: Access-Control-Request-Method
<span class="hljs-attribute">Vary</span>: Access-Control-Request-Headers
<span class="hljs-attribute">Content-Type</span>: application/json
<span class="hljs-attribute">Transfer-Encoding</span>: chunked
<span class="hljs-attribute">Date</span>: Tue, 13 Sep 2022 11:08:29 GMT
<span class="hljs-attribute">Keep-Alive</span>: timeout=60
<span class="hljs-attribute">Connection</span>: keep-alive
<span class="hljs-attribute">Content-Length</span>: 1087

<span class="json">{
  <span class="hljs-attr">"code"</span> : <span class="hljs-string">"T-S008"</span>,
  <span class="hljs-attr">"message"</span> : <span class="hljs-string">"가게 리뷰 조회에 성공했습니다"</span>,
  <span class="hljs-attr">"data"</span> : {
    <span class="hljs-attr">"reviews"</span> : [ {
      <span class="hljs-attr">"user"</span> : {
        <span class="hljs-attr">"userId"</span> : <span class="hljs-number">3</span>,
        <span class="hljs-attr">"userNickname"</span> : <span class="hljs-string">"포키"</span>,
        <span class="hljs-attr">"userProfileImage"</span> : <span class="hljs-string">"https://s3.ap-northeast-2.amazonaws.com/user/forky.jpg"</span>
      },
      <span class="hljs-attr">"store"</span> : {
        <span class="hljs-attr">"storeId"</span> : <span class="hljs-number">1</span>,
        <span class="hljs-attr">"storeName"</span> : <span class="hljs-string">"맛있는가게"</span>,
        <span class="hljs-attr">"addressName"</span> : <span class="hljs-string">"서울시 송파구 11-22"</span>,
        <span class="hljs-attr">"roadAddressName"</span> : <span class="hljs-string">"서울시 송파구 좋은길1"</span>
      },
      <span class="hljs-attr">"like"</span> : {
        <span class="hljs-attr">"count"</span> : <span class="hljs-number">0</span>,
        <span class="hljs-attr">"userLikeStatus"</span> : <span class="hljs-literal">false</span>
      },
      <span class="hljs-attr">"reviewId"</span> : <span class="hljs-number">34</span>,
      <span class="hljs-attr">"reviewContent"</span> : <span class="hljs-string">"전체 공개된 리뷰 게시물 29"</span>,
      <span class="hljs-attr">"reviewScore"</span> : <span class="hljs-number">3.0</span>,
      <span class="hljs-attr">"reviewVisibility"</span> : <span class="hljs-string">"PROTECTED"</span>,
      <span class="hljs-attr">"visitedDate"</span> : <span class="hljs-string">"2022-01-01"</span>,
      <span class="hljs-attr">"tags"</span> : [ <span class="hljs-string">"tag1"</span>, <span class="hljs-string">"tag2"</span> ],
      <span class="hljs-attr">"images"</span> : [ <span class="hljs-string">"https://s3.ap-northeast-2.amazonaws.com/sikdorak/test.jpg"</span> ],
      <span class="hljs-attr">"createdAt"</span> : <span class="hljs-string">"2022-09-13T20:08:29.613803"</span>,
      <span class="hljs-attr">"updatedAt"</span> : <span class="hljs-string">"2022-09-13T20:08:29.613803"</span>
    } ],
    <span class="hljs-attr">"page"</span> : {
      <span class="hljs-attr">"size"</span> : <span class="hljs-number">1</span>,
      <span class="hljs-attr">"prev"</span> : <span class="hljs-number">0</span>,
      <span class="hljs-attr">"next"</span> : <span class="hljs-number">33</span>,
      <span class="hljs-attr">"last"</span> : <span class="hljs-literal">false</span>
    }
  }
}</span></code></pre>
</div>
</div>
</div>
<div class="sect4">
<h5 id="_response_24_response_fields"><a class="link" href="#_response_24_response_fields">Response fields</a></h5>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 25%;">
<col style="width: 25%;">
<col style="width: 25%;">
<col style="width: 25%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top">필드명</th>
<th class="tableblock halign-left valign-top">데이터 타입</th>
<th class="tableblock halign-left valign-top">설명</th>
<th class="tableblock halign-left valign-top">제약 사항</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>code</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>String</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">식도락 서버 전용 상태코드</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>message</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>String</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">자세한 응답 메세지</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>data.reviews[].user</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>Object</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">유저 정보</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>data.reviews[].user.userId</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>Number</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">유저 아이디</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">· Must not be null<br></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>data.reviews[].user.userNickname</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>String</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">유저 이름</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">· Must not be null<br>
· Size must be between 1 and 30 inclusive<br></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>data.reviews[].user.userProfileImage</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>String</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">유저 프로필 이미지</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">· Must be a well-formed URL<br></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>data.reviews[].store</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>Object</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">가게 정보</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>data.reviews[].store.storeId</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>Number</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">가게 아이디</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">· Must not be null<br></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>data.reviews[].store.storeName</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>String</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">가게 이름</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">· Must not be blank<br>
· Size must be between 1 and 500 inclusive<br></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>data.reviews[].store.addressName</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>String</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">지번 주소</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>data.reviews[].store.roadAddressName</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>String</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">도로명 주소</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>data.reviews[].like</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>Object</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">좋아요 정보</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>data.reviews[].like.count</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>Number</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">좋아요 개수</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">· Must be at least 0<br>
· Must not be null<br></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>data.reviews[].like.userLikeStatus</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>Boolean</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">유저의 좋아요 여부</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">· Must not be null<br></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>data.reviews[].reviewId</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>Number</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">리뷰 아이디</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">· Must not be null<br></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>data.reviews[].reviewContent</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>String</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">리뷰 내용</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">· Must not be blank<br>
· Size must be between 1 and 500 inclusive<br></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>data.reviews[].reviewScore</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>Number</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">리뷰 점수</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">· Must match the regular expression <code>1-5</code><br>
· Must not be null<br></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>data.reviews[].reviewVisibility</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>String</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">리뷰 게시물의 공개 범위</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">· Must match the regular expression <code>public|protected|private</code><br>
· Must not be null<br></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>data.reviews[].visitedDate</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>String</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">가게 방문일</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">· Must be in the past or the present<br>
· Must match the regular expression <code>yyyy-MM-dd</code><br>
· Must not be null<br></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>data.reviews[].tags</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>Array</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">리뷰를 표현하는 태그들</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">· Size must be between 0 and 30 inclusive<br></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>data.reviews[].images</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>Array</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">리뷰를 위한 사진 URL</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">· Must be a well-formed URL<br>
· Size must be between 0 and 10 inclusive<br></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>data.reviews[].createdAt</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>String</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">리뷰 생성 시간</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">· Must match the regular expression <code>yyyy-MM-dd</code><br>
· Must not be null<br></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>data.reviews[].updatedAt</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>String</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">리뷰 수정 시간</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">· Must match the regular expression <code>yyyy-MM-dd</code><br>
· Must not be null<br></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>data.page.size</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>Number</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">페이지 크기</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">· Must be at least 0<br>
· Must be at most 20<br>
· Must not be null<br></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>data.page.prev</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>Number</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">이전 페이지를 가리키는 커서</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">· Must be at least 0<br>
· Must be at most 9223372036854775807<br>
· Must not be null<br></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>data.page.next</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>Number</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">다음 페이지를 가리키는 커서</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">· Must be at least 0<br>
· Must be at most 9223372036854775807<br>
· Must not be null<br></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>data.page.last</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><code>Boolean</code></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">마지막 페이지 여부</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</details>

[SDR-273]: https://jjikmuk.atlassian.net/browse/SDR-273?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: be/src/docs/asciidoc/store/store.adoc [hunk 1]
```diff
@@ -88,13 +88,13 @@ API : `PUT /api/stores`
...
 
-operation::store_verify_or_save_documentation_test/search_store_by_name_containing_success[snippets='http-request,request-body']
+operation:store_verify_or_save_documentation_test/search_store_by_name_containing_success[snippets='http-request,request-body']
 
 ==== `Response`
 
 operation::store_verify_or_save_documentation_test/search_store_by_name_containing_success[snippets='http-response,response-fields']
 
-=== 가게 상세 조회
+=== 가게 상세 조회`
```
Comment (ku-kim) [inline be/src/docs/asciidoc/store/store.adoc]: api 문서는 다음 PR에 싹 바꾸겠습니다 `_'

File: be/src/docs/asciidoc/store/store.adoc [hunk 2]
```diff
@@ -88,13 +88,13 @@ API : `PUT /api/stores`
 
 ==== `Request`
 
-operation::store_verify_or_save_documentation_test/search_store_by_name_containing_success[snippets='http-request,request-body']
+operation:store_verify_or_save_documentation_test/search_store_by_name_containing_success[snippets='http-request,request-body']
```
Comment (ku-kim) [inline be/src/docs/asciidoc/store/store.adoc]: ":"가 한개라면 작동 안할 거 같네요.! 
머지된이후 제가 수정할게요

File: be/src/test/java/com/jjikmuk/sikdorak/documentationtest/store/StoreReviewsSearchByStoreIdDocumentationTest.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+import com.jjikmuk.sikdorak.common.ResponseCodeAndMessages;
+import com.jjikmuk.sikdorak.documentationtest.InitDocumentationTest;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+
+@DisplayName("문서화 : Store(특정 가게)의 리뷰 리스트 조회")
```
Comment (ku-kim) [inline be/src/test/java/com/jjikmuk/sikdorak/documentationtest/store/StoreReviewsSearchByStoreIdDocumentationTest.java:18]: 통일성 good!

File: be/src/test/java/com/jjikmuk/sikdorak/integration/review/ReviewsSearchByStoreIdIntegrationTest.java [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+            // when
+            ReviewListResponse reviewResponse = reviewService.searchReviewsByStoreId(
+                store.getId(), pageRequest);
+
+            // then
+            assertThat(reviewResponse).isNotNull();
+
+            List<ReviewDetailResponse> reviews = reviewResponse.reviews();
+            assertThat(reviews).isNotEmpty()
```
Comment (ku-kim) [inline be/src/test/java/com/jjikmuk/sikdorak/integration/review/ReviewsSearchByStoreIdIntegrationTest.java:46]: reviews에서 isNotEmpty() 체크하니 assertThat(reviewResponse).isNotNull(); 은 빠져도 괜찮지 않을까요?!

File: be/src/test/java/com/jjikmuk/sikdorak/documentationtest/store/StoreReviewsSearchByStoreIdDocumentationTest.java [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+				document(DEFAULT_RESTDOC_PATH,
+					STORE_SEARCH_REVIEWS_REQUEST_PARAM_SNIPPET,
+					STORE_SEARCH_REVIEWS_PAGING_REQUEST_PARAM_SNIPPET,
+					STORE_SEARCH_REVIEWS_RESPONSE_SNIPPET))
+			.accept(MediaType.APPLICATION_JSON_VALUE)
+			.header("Content-type", "application/json")
+			.param("after", 0)
+			.param("size", 1)
+
+			.when()
```
Comment (ku-kim) [inline be/src/test/java/com/jjikmuk/sikdorak/documentationtest/store/StoreReviewsSearchByStoreIdDocumentationTest.java]: p3; when, then 개행 통일하면 좋을거같아요

File: be/src/main/java/com/jjikmuk/sikdorak/review/repository/ReviewRepository.java [hunk 6]
```diff
@@ -69,4 +69,14 @@ SELECT avg(r.reviewScore.reviewScore) FROM Review r
          WHERE r.storeId = :storeId AND r.deleted = false""")
     Optional<Double> findReviewScoreAverageByStoreId(@Param("storeId") long storeId);
 
+    @Query("""
+        SELECT r FROM Review r
+         WHERE r.storeId = :storeId
+           AND r.id <= :targetId
+         ORDER BY r.id DESC
+        """)
```
Comment (ku-kim) [inline be/src/main/java/com/jjikmuk/sikdorak/review/repository/ReviewRepository.java]: p1; 쿼리가 public, protected, private 모두 조회될 거 같은데요!?

이야기 나눴던, public만 조회하는건 어떠신가요?

```suggestion
    @Query("""
        SELECT r FROM Review r
         WHERE r.storeId = :storeId
           AND r.reviewVisibility = 'PUBLIC'
           AND r.id <= :targetId
         ORDER BY r.id DESC
        """)
```

File: be/src/test/java/com/jjikmuk/sikdorak/integration/review/ReviewsSearchByStoreIdIntegrationTest.java [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+                .allSatisfy(reviewDetail ->
+                    assertAll(
+                        () -> assertThat(reviewDetail.store().storeId()).isEqualTo(store.getId()),
+                        () -> assertThat(reviewDetail.store().storeName()).isEqualTo(
+                            store.getStoreName())
+                    )
```
Comment (ku-kim) [inline be/src/test/java/com/jjikmuk/sikdorak/integration/review/ReviewsSearchByStoreIdIntegrationTest.java]: PUBLIC한 리뷰들만 조회된다로, 접근 권한 여부도 테스트파면 좋을거같아요!

**Review by ku-kim — CHANGES_REQUESTED** [review]
고생 많으셨어요 제이!

+a) API 문서는 만들어주신 토대로 다음 PR에서 제가 수정하도록할게요

File: be/src/docs/asciidoc/store/store.adoc (hunk 1 above)
Comment (jinan159) [reply be/src/docs/asciidoc/store/store.adoc]: 제가 수정하겠습니다ㅎㅎ;;

File: be/src/docs/asciidoc/store/store.adoc (hunk 2 above)
Comment (jinan159) [reply be/src/docs/asciidoc/store/store.adoc]: 이 부분도 같이 수정할게요

**Review by seokho-ham — DISMISSED** [review]
고생하셨어요 제이~👍
(이미 쿠킴이 필요한 부분들은 이야기를 하셔서 따로 코멘트는 남기지 않았습니다!)

File: be/src/test/java/com/jjikmuk/sikdorak/documentationtest/store/StoreReviewsSearchByStoreIdDocumentationTest.java (hunk 5 above)
Comment (jinan159) [reply be/src/test/java/com/jjikmuk/sikdorak/documentationtest/store/StoreReviewsSearchByStoreIdDocumentationTest.java]: 아.. 이게 제가 IntelliJ 에서 commit 할때 code reformat 하도록 설정되어있어서 그런가보네요;;
해제하고 수정 해놓겠습니다

File: be/src/main/java/com/jjikmuk/sikdorak/review/repository/ReviewRepository.java (hunk 6 above)
Comment (jinan159) [reply be/src/main/java/com/jjikmuk/sikdorak/review/repository/ReviewRepository.java]: 추가한다는게 깜빡했네요;; 수정해서 push 했습니다

File: be/src/test/java/com/jjikmuk/sikdorak/integration/review/ReviewsSearchByStoreIdIntegrationTest.java (hunk 7 above)
Comment (jinan159) [reply be/src/test/java/com/jjikmuk/sikdorak/integration/review/ReviewsSearchByStoreIdIntegrationTest.java]: 반영완료👍

File: be/src/test/java/com/jjikmuk/sikdorak/integration/review/ReviewsSearchByStoreIdIntegrationTest.java (hunk 4 above)
Comment (jinan159) [reply be/src/test/java/com/jjikmuk/sikdorak/integration/review/ReviewsSearchByStoreIdIntegrationTest.java:46]: null 이면 NPE 가 터질테니 그것도 괜찮은 방법이긴 하네요..!
그런데 저는 NPE 터지는걸 잡는거보다 assertThat 으로 잡는것도 괜찮을 거 같습니다 ㅎㅎ

**Review by ku-kim — APPROVED** [review]
고생 넘 많으셨어요 jay!!! 🥹

**Pull request merged**